lib/core/src/sdk.rs

```diff
@@ -1701,10 +1707,16 @@ impl LiquidSdk {
         let receiver_amount_sat = user_lockup_amount_sat - fees_sat;
         let claim_address = self.onchain_wallet.next_unused_address().await?.to_string();
 
+        // Reserve this address until the timeout block height
```

Comment:
For chain swaps, I'm 99% sure that claims (coop / non-coop) can happen even after the timeout block height. If the user locks up funds on time (i.e. before the timeout block), the swap won't be marked as expired and it can be claimed even after the timeout block.

I'm checking with Boltz to confirm.

If this is true, it would mean the claim LBTC address could still be used by this swap even after it's un-reserved.

Reply:
Looks like I was wrong, claiming is only possible until the timeout block height. After that only refunds are allowed.

Reply:
Thanks for verifying @ok300